sample_configs/sample_config.yml

```diff
@@ -1,39 +1,27 @@
-liquid_hostname: liquidnode.liquid # defaults to liquidnode.liquid
-
 network:
-  hotspot: # hotspot is always on (Primary node WiFi/'admin' hotspot interface)
-    ssid: foo
-    password: bar
-    dhcp_server:
-      ip: 10.0.0.1 # defaults to 10.0.0.1
-      netmask: 255.255.255.0 # defaults to 255.255.255.0
-      dhcp_range: some_magic_dnsmasq_string? # defaults to 10.0.0.100-255
-  
-  lan: # for wired config of dhcp/dns server (Primary node)
-    interface: ethernet | none # no default
-    mode: dhcp_server | static # default to dhcp client
-    dhcp_server:
-      ip: 10.0.0.1 # defaults to 10.0.0.1
-      netmask: 255.255.255.0 # defaults to 255.255.255.0
-      dhcp_range: some_magic_dnsmasq_string? # defaults to 10.0.0.100-255
-    static:
-      ip: 192.168.66.66  # no default
-      netmask: 255.255.255.0 # no default
-      gateway: 192.168.66.1 # no default
-      dns_server: 192.168.66.1 # no default
-      
-  wan: # (Secondary node config or connection to internet)
-    interface: ethernet | wifi | none # no default
-    wifi:
+  domain: liquidnode.liquid # defaults to liquidnode.liquid
```

Comment:
We shouldn't have a default for this.

Reply:
Eh, I think we should have sane defaults for everything, including hostpot ssid and password. If you want to quickly demo the box, or play with it, it should be easy.

Reply:
So you're saying an empty config should work.

Reply:
Yep.